build_library/generate_au_zip.py

```diff
@@ -153,7 +155,7 @@ def CopyRequiredFiles(dest_files_root, allow_list):
     sys.exit(1)
 
   all_files = DYNAMIC_EXECUTABLES + STATIC_FILES
-  all_files = map(os.path.expanduser, all_files)
+  all_files = list(map(os.path.expanduser, all_files))
```

Comment:
in line 166 (can't select in the review): is that a lonely `all_files` statement?

Reply:
Huh, seems to be. I'll drop it.

Reply:
Yeah! seems like so!